src/routes/(console)/bottomAlerts.ts

```diff
@@ -9,41 +13,133 @@ import { isCloud } from '$lib/system';
-    });
+            // Day 2 = Tuesday 3 PM CET
+            show: isPromoLive('2025-05-20', '15:00')
+        },
+        {
+            id: 'modal:init3_keys',
+            src: getSrc(Init3KeysPromo),
+            title: 'Announcing: Dev Keys',
+            message:
+                'A new Appwrite feature that lets you bypass rate limits during local development.',
+            plan: 'free',
+            importance: 8,
+            scope: 'everywhere',
+            cta: sharedCTA('https://apwr.dev/devkeys'),
+
+            // Day 3 = Wednesday 3 PM CET
+            show: isPromoLive('2025-05-21', '15:00')
+        },
+        {
+            id: 'modal:init3_images',
+            src: getSrc(Init3ImagesPromo),
+            title: 'Announcing: New image formats',
+            message:
+                'We have added support for two new image formats in Appwrite Storage: HEIC and AVIF.',
+            plan: 'free',
+            importance: 8,
+            scope: 'everywhere',
+            cta: sharedCTA('https://apwr.dev/imagefo'),
+
+            // Day 4 = Thursday 3 PM CET
+            show: isPromoLive('2025-05-22', '15:00')
+        },
+        {
+            id: 'modal:init3_file_tokens',
+            src: getSrc(Init3TokensPromo),
+            title: 'Announcing: File Tokens',
+            message:
+                'File tokens let you share files easily and securely, without modifying permissions or changing project access.',
+            plan: 'free',
+            importance: 8,
+            scope: 'everywhere',
+            cta: sharedCTA('https://apwr.dev/filet'),
+
+            // Day 5 = Friday 3 PM CET
+            show: isPromoLive('2025-05-23', '15:00')
+        }
+    ];
+
+    const currentPromo = promos.find((promo) => promo.show);
+
+    if (currentPromo) {
+        listOfPromotions.push(currentPromo);
+
+        setMobileSingleAlertLayout({
+            title: currentPromo.title,
+            message: currentPromo.message,
+            enabled: true,
+            cta: currentPromo.cta
+        });
+    }
 }
 
 export function addBottomModalAlerts() {
     listOfPromotions.forEach((promotion) => showBottomModalAlert(promotion));
 }
+
+function isPromoLive(date: string, time: string, timeZone: string = 'Europe/Paris'): boolean {
+    const isoString = `${date}T${time}:00`;
+
+    const formatter = new Intl.DateTimeFormat('en', {
+        timeZone,
+        year: 'numeric',
+        month: '2-digit',
+        day: '2-digit',
+        hour: '2-digit',
+        minute: '2-digit',
+        hourCycle: 'h23'
+    });
+
+    const parts = formatter.formatToParts(new Date(isoString));
+    const map = Object.fromEntries(
+        parts.filter((p) => p.type !== 'literal').map((p) => [p.type, p.value])
+    );
+
+    const target = new Date(`${map.year}-${map.month}-${map.day}T${map.hour}:${map.minute}:00`);
+
+    const now = new Date();
+    const isSameDay = now.toISOString().slice(0, 10) === target.toISOString().slice(0, 10);
```

Comment:
if fell like this is a bit overkill? cant we just do one date object to be passed here and compare?